src/components/ha-data-table.ts

```diff
@@ -427,7 +427,7 @@ export class HaDataTable extends BaseElement {
       }
 
       .mdc-data-table {
-        background-color: var(--card-background-color);
+        background-color: var(--paper-card-background-color);
```

Comment:
revert this, `card-background-color` is what we use for `ha-card`. `paper` is a propietary var

Reply:
We use `paper-card-background-color` that is derived from `card-background-color` in `ha-card`. Most themes set `paper-card-background-color` but I agree that should be `card-background-color`

Reply:
I think that we have been changing var to fallback to paper-card-background-color, but prefer card-background-color. Themes should really be updated.